apps/sasquatch/google-services.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
Why do you check in this file? I know this is mandatory but we used to have a local copy of this one.